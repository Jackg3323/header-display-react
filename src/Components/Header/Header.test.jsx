import renderer from "react-test-renderer";
import Header from "./Header";

it("renders correctly", () => {
  const tree = renderer.create(<Header name="Mark" />).toJSON();
  expect(tree).toMatchSnapshot();
});
