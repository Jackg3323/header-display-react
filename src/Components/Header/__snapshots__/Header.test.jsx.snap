// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<h1
  className="my-8 text-center text-3xl"
  label="title"
>
  Oh, Hi 
  Mark
</h1>
`;
